superset/dao/base.py

```diff
@@ -75,7 +75,7 @@ def find_by_ids(cls, model_ids: List[int]) -> List[Model]:
         return query.all()
 
     @classmethod
-    def create(cls, properties: Dict, commit=True) -> Optional[Model]:
+    def create(cls, properties: Dict, commit=True) -> Model:
```

Comment:
A bit redundant but since `check_ownership` also adds `bool` for the named parameter:
`def create(cls, properties: Dict, commit: bool = True) -> Model:`